general/models/action.py

```diff
@@ -43,6 +44,7 @@ class ActionObjectType:
     QUESTION = "Question"
     SUBMISSION = "Submission"
     USER = "User"
+    TEAM = "Team"
```

Comment:
These require a migration. Did you not face an error?

Reply:
It didn't give me any errors.
I ran the migration command, but I still have this:

![image](https://user-images.githubusercontent.com/60047943/212982216-044d5ec1-4e96-4c68-b7b8-b6ce752ee6dc.png)

Reply:
Have you created the migrations?
you have to run `python manage.py createmigrations` first and then `python manage.py migrate`.